### ISSUE TITLE
Low contrast of colored texts from `dayfox`

### ISSUE THREAD

**Issue by oxalica** [EdenEast/nightfox.nvim#282]
## Description

`dayfox`'s colored texts are much harder to read than `nightfox`.
All primary colors from `dayfox` except black failed the contrast requirement of [WCAG AA level](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast).
Example of red v.s. background: https://webaim.org/resources/contrastchecker/?fcolor=B95D76&bcolor=EAEAEA

## Behavior

**Expected behavior**
Colored texts using `dayfox` should be as easy to read as `nightfox`.

**Screenshots**
[README's picture](https://github.com/EdenEast/nightfox.nvim/blob/21e7e733e28b79b876e6bd514e6046a8da63ddae/readme.md#dayfox) seems convincing enough.

**Comment by EdenEast** [comment]
I do agree that dayfox does need some updates to make it more contrasting. I will try and do a pass over it to make it more contrasting.

**Comment by oxalica** [comment]
BTW, there are also colors in other themes having low contrasts, including [red v.s. background](https://webaim.org/resources/contrastchecker/?fcolor=738091&bcolor=192330) and [comment v.s. background](https://webaim.org/resources/contrastchecker/?fcolor=C94F6D&bcolor=192330) in `nightfox`.

**Comment by D00mch** [comment]
I preferred the previous dayfox. Please do not update "dawnfox":) I'm now using it.

**Comment by EdenEast** [comment]
For those that want to use the old dayfox you can use tag [v2.1.0](https://github.com/EdenEast/nightfox.nvim/tree/v2.1.0).

**Comment by EdenEast** [comment]
For those that want to use the old dayfox check this [comment](https://github.com/EdenEast/nightfox.nvim/pull/285#issuecomment-1373841637) on how to override it back.
